packages/terra-detail-view/examples/DetailViewDivided.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Grid from 'terra-grid';
+import DetailView from '../src/DetailView';
+
+const DetailViewDivided = () => (
+  <DetailView
+    title={<div className="terra-Title terra-Title--large">Header</div>}
```

Comment:
We should be using heading elements for titles. Can you update the div to a h(1-6) element.
e.g. `title={<h1 className="terra-Title terra-Title--large">Header</h1>}